common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -114,6 +114,18 @@ class CapaFields(object):
         help=_("Amount of time after the due date that submissions will be accepted"),
         scope=Scope.settings
     )
+    show_correctness = String(
+        display_name=_("Show Results"),
+        help=_("Defines when to show whether a learner's answer to the problem is correct. "
+               "Configured on the subsection. A default value for the course can be set in Advanced Settings."),
```

Comment:
@pomegranited This setting does not show in Advanced Settings, and I verified with @sstack22 that we don't want it to.

It doesn't show in Advanced Settings because you added it to FILTERED_LIST in course_metadata. The purpose of FILTERED_LIST is to filter out settings that should not be editable in Advanced Settings. I personally don't see it in Advanced Settings-- if somehow you are seeing it, please let me know.